docs/design/game-design.md

```
@@ -37,6 +44,11 @@ status: active
 - **Two-finger drag** → camera pan
 - **Pinch** → zoom
 
+### Layout Adaptation
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)